composer.json

```diff
@@ -6,21 +6,21 @@
         "ext-ctype": "*",
         "ext-iconv": "*",
         "aws/aws-sdk-php": "^3.71",
-        "gaufrette/extras": "^0.1.0",
-        "knplabs/knp-gaufrette-bundle": "^0.5.3",
-        "ramsey/uuid": "^3.8",
-        "ramsey/uuid": "^3.8",
         "doctrine/annotations": "^1.6",
         "friendsofsymfony/jsrouting-bundle": "^2.2",
         "fzaninotto/faker": "^1.8",
+        "gaufrette/extras": "^0.1.0",
```

Comment:
Could you please try to move these lines back, the only line that should be added here is `"symfony/security-bundle": "*",`